include/lsst/afw/detection/Psf.h

```diff
@@ -283,13 +344,11 @@ class Psf : public daf::base::Citizen,
     virtual geom::Box2I doComputeBBox(geom::Point2D const& position, image::Color const& color) const = 0;
     //@}
 
+
     bool const _isFixed;
-    mutable std::shared_ptr<Image> _cachedImage;
-    mutable std::shared_ptr<Image> _cachedKernelImage;
-    mutable image::Color _cachedImageColor;
-    mutable image::Color _cachedKernelImageColor;
-    mutable geom::Point2D _cachedImagePosition;
-    mutable geom::Point2D _cachedKernelImagePosition;
+    typedef utils::Cache<detail::PsfCacheKey, std::shared_ptr<Image>> PsfCache;
```

Comment:
`using PsfCache = utils::Cache<detail::PsfCacheKey, std::shared_ptr<Image>>` is preferred in C++11.